doc/integrator/make.rst

```diff
@@ -6,21 +6,21 @@ Build configuration
 Env files
 ---------
 
-Usually we have the following env files::
+Usually we have the following env files:
 
- - ``env.default`` for the c2cgeoportal default configuration
- - ``env.project`` for the project configuration
- - ``env.<town>`` in a multi instance project we will have sone configuration per town
- - ``env.(dev|int|prod)`` we usualy have a file per integations environment,
-   not needed for OpenShift project
+- ``env.default`` for the c2cgeoportal default configuration
+- ``env.project`` for the project configuration
+- ``env.<town>`` in a multi instance project we will have sone configuration per town
```

Comment:
I volontary avoid the instance therm because instance sometimes mean town like here, sometimes for int-prod, ...

Reply:
Town is strange, "municipality" would be more general. But the instances are not necessarily municipalities, in the case of Rudaz for example, it is simply customers of Rudaz, like "cave des vins XYZ". 

If we speak about multi-instance projects, than the thing that is "multi" is an "instance", by definition. Otherwise, we have to find another term than "multi-instance".